glance/templates/api/configmaps/glance-api.conf.yaml.j2

```diff
@@ -40,14 +40,25 @@ data:
     [glance_store]
     filesystem_store_datadir = /var/lib/glance/images/
 
-    {% if not ceph.enabled -%}
-    stores = file, http
-    default_store = file
-    {%- else -%}
+    {% if ceph.enabled -%}
     stores = file, http, rbd
     default_store = rbd
     rbd_store_pool = {{ ceph.glance_pool }}
     rbd_store_user = {{ ceph.glance_user }}
     rbd_store_ceph_conf = /etc/ceph/ceph.conf
     rbd_store_chunk_size = 8
+
+    {% elif swift.enabled -%}
+    stores = file, http, swift
```

Comment:
I believe it calls for an additional flag (with swift generally being ok as default).